### ISSUE TITLE
slow_parameter_passing false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3023]
```abap
CLASS lcl DEFINITION.
  PUBLIC SECTION.
    TYPES ty TYPE STANDARD TABLE OF i WITH EMPTY KEY.
    METHODS foo IMPORTING VALUE(tab) TYPE ty.
ENDCLASS.
CLASS lcl IMPLEMENTATION.
  METHOD foo.
    LOOP AT tab ASSIGNING FIELD-SYMBOL(<sdf>).
      <sdf> = 2.
    ENDLOOP.
  ENDMETHOD.
ENDCLASS.
```

![image](https://github.com/abaplint/abaplint/assets/5888506/f8544c84-b20f-4ad8-ba7b-c6d688aca474)
